cfg_example.yml

```diff
@@ -0,0 +1,16 @@
+# It's an example of the configuration file for the Censor.
+parser:
+  # If true, the parser will use the json tag name instead of the struct field name.
+  # In case of absence of json tag, the parser will use the struct field name.
+  use-json-tag-name: false
+formatter:
+  # Given string will be used as a mask for sensitive data.
+  mask-value: "[CENSORED]"
+  # If true, '&' (a pointer symbol) will be displayed for pointer types before the pointed value.
+  display-pointer-symbol: false
+  # If true, the formatter will display the struct name (including the last part of the package name).
+  display-struct-name: false
+  # If true, the formatter will display the map type.
+  display-map-type: false
```

Comment:
Maybe it makes sense to add that this field is only applicable for text representation?